python/mathsys/dev/builder/code.py

```
@@ -0,0 +1,75 @@
+        finally:
+            os.remove(filename)
+            os.remove(ir)
+    #~ CLASS -> COMMAND CREATOR HELPER
+    def command(self, target: str, filename: str) -> list[str]:
+        return [
+            "rustc",
+            "+nightly",
+            "../bin/main.rs",
+            "--target", TARGETS[target],
+            "--sysroot", subprocess.check_output(
+                ["rustc", "+nightly", "--print", "sysroot"],
```

Ruff (0.14.6)

Starting a process with a partial executable path

(S607)